src/CTA.Rules.ProjectFile/ProjectFileCreator.cs

```diff
@@ -49,6 +49,17 @@ public class ProjectFileCreator
 {2}
 {3}
 {4}
+</Project>";
+
+        private readonly string _csCoreProjExe =
+@"<Project Sdk=""{0}"">
+  <PropertyGroup>
+    <TargetFramework>{1}</TargetFramework>
+    <OutputType>Exe</OutputType>
```

Comment:
Does this apply to every WCF project we support?

Reply:
Yes in the current case we want the WCF Service Projects to be Executable